plotly/offline/offline.py

```diff
@@ -171,6 +184,34 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
                            height=height, width=width)))
 
 
+def iplot_mpl(mpl_fig,mpl_to_plotly_kw={},iplot_kw={}):
+    '''
+    Convert a matplotlib figure to plotly dictionary  plot inside an 
+    IPython notebook without connecting to an external server.
+    '''
+    plotly_plot = tools.mpl_to_plotly(mpl_fig,**mpl_to_plotly_kw)
+    return iplot(plotly_plot,**iplot_kw)
+
```

Comment:
little :cow2: to keep with the consistency of this file: add a line here (so there are 2 lines between defs)